File: GeeksforGeeks/Sept_2024/Week_1/unionOfTwoArray.js
//to find unique elements in both the array and then print how many element it has
class Solution {
    // Function to return the count of number of elements in the union of two arrays.
    doUnion(arr1, arr2) {
        // code here
       const newArray = [...arr1,...arr2];
       
       const set1 = new Set(newArray);
       
       return set1.size;
    }
}

File: GeeksforGeeks/Sept_2024/unionoftwoArray.java
class Solution {
    public static int doUnion(int arr1[], int arr2[]) {
        // Your code here
        Set<Integer> union = new HashSet<>();
        
        for(int i=0;i<arr1.length;i++){
            union.add(arr1[i]);
        }
        for(int i=0;i<arr2.length;i++){
            union.add(arr2[i]);
        }
    return union.size();
    }
}